src/pipelinedefinition.cpp

```diff
@@ -1053,17 +1053,17 @@ const tensor_map_t PipelineDefinition::getOutputsInfo() const {
 }
 
 std::shared_ptr<TensorInfo> applyDemultiplexerShapeForTensor(const std::shared_ptr<TensorInfo>& tensorInfo, uint32_t demultiplyCount) {
```

Comment:
Here you use `DYNAMIC_DIMENSION` whould should be hidden. How about changing `createCopyWithEffectiveDimensionPrefix` to use `Dimension` and this way we can use `Dimension::any` which is designed for this purpose?

Reply:
This comes from NodeInfo std::optional<size_t> demultiplyCount;

Reply:
This could be resolved if -1 would be supported as dynamic demultiplexer/dimension in DAG.